docs/examples/use_cases/pytorch/efficientnet/image_classification/dataloaders.py

```diff
@@ -274,6 +274,190 @@ def expand(num_classes, dtype, tensor):
+
+    def __iter__(self):
+        if self.dataloader.sampler is not None and isinstance(
+            self.dataloader.sampler,
+            torch.utils.data.distributed.DistributedSampler,
+        ):
+
+            self.dataloader.sampler.set_epoch(self.epoch)
+        self.epoch += 1
+        return PrefetchedWrapper.prefetched_loader(
+            self.dataloader, self.num_classes, self.one_hot
+        )
+
+    def __len__(self):
+        return len(self.dataloader)
+
+
+def get_pytorch_train_loader(
+    data_path,
+    image_size,
+    batch_size,
+    num_classes,
+    one_hot,
+    interpolation="bilinear",
+    augmentation=None,
+    start_epoch=0,
+    workers=5,
+    _worker_init_fn=None,
+    prefetch_factor=2,
+    memory_format=torch.contiguous_format,
+):
+    interpolation = {"bicubic": Image.BICUBIC, "bilinear": Image.BILINEAR}[
+        interpolation
+    ]
+    traindir = os.path.join(data_path, "train")
+    transforms_list = [
+        transforms.RandomResizedCrop(image_size, interpolation=interpolation),
+        transforms.RandomHorizontalFlip(),
+    ]
+    if augmentation == "disabled":
+        pass
+    elif augmentation == "autoaugment":
+        transforms_list.append(AutoaugmentImageNetPolicy())
+    else:
+        raise NotImplementedError(
+            f"Automatic augmentation: '{augmentation}' is not supported"
+            " for PyTorch data loader."
+        )
+
+    transforms_list.append(transforms.ToTensor())
+    transforms_list.append(
+        transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])
+    )
+    train_dataset = datasets.ImageFolder(
+        traindir, transforms.Compose(transforms_list)
+    )
+
+    if torch.distributed.is_initialized():
+        train_sampler = torch.utils.data.distributed.DistributedSampler(
+            train_dataset, shuffle=True
+        )
+    else:
+        train_sampler = None
+
+    train_loader = torch.utils.data.DataLoader(
+        train_dataset,
+        sampler=train_sampler,
+        batch_size=batch_size,
+        shuffle=(train_sampler is None),
+        num_workers=workers,
+        worker_init_fn=_worker_init_fn,
+        pin_memory=True,
+        collate_fn=partial(fast_collate, memory_format),
+        drop_last=True,
+        persistent_workers=True,
+        prefetch_factor=prefetch_factor,
+    )
+
+    return (
+        PrefetchedWrapper(
+            train_loader,
+            start_epoch,
+            num_classes,
+            one_hot,
+        ),
+        len(train_loader),
+    )
+
+
+def get_pytorch_val_loader(
+    data_path,
+    image_size,
+    batch_size,
+    num_classes,
+    one_hot,
+    interpolation="bilinear",
+    workers=5,
+    _worker_init_fn=None,
+    crop_padding=32,
+    memory_format=torch.contiguous_format,
+    prefetch_factor=2,
+):
+    interpolation = {"bicubic": Image.BICUBIC, "bilinear": Image.BILINEAR}[
+        interpolation
+    ]
+    valdir = os.path.join(data_path, "val")
+    transforms_list = [
```

Comment:
Nitpick:
```suggestion
    transform_list = [
```
or
```suggestion
    transforms = [
```
(but the latter would cause a name collision with the imported module).

Reply:
Fixed